port/port.h

```diff
@@ -15,6 +15,22 @@
 #define __dead __attribute__((__noreturn__))
 #endif
 
+#ifndef __BEGIN_DECLS
+#define __BEGIN_DECLS
+#endif
+
+#ifndef __END_DECLS
+#define __END_DECLS
+#endif
+
+#ifndef MIN
+#define MIN(a,b) ((a) < (b) ? (a) : (b))
```

Comment:
@zoulasc when `sys/param.h` gets included (in lib/bl.c) these definitions get redefined.
I had to `#include <sys/param.h>` in this file in order to avoid it.

Reply:
Weird, those are defined in <sys/cdefs.h> and <sys/param.h> should only include <sys/cdefs.h> only if _KERNEL is defined?

Reply:
Ah, you mean MIN/MAX probably, not __BEGIN_DECLS...